packages/core/src/components/axes/zoom-bar.ts

```diff
@@ -290,7 +313,12 @@ export class ZoomBar extends Component {
 			.attr("width", handleWidth)
 			.attr("height", handleHeight)
 			.attr("cursor", "pointer")
-			.style("display", null); // always display
+			.style(
+				"display",
+				isDataLoading || isNaN(selection[0]) || isNaN(selection[1])
+					? "none"
+					: null
```

Comment:
maybe use a variable like
```
const displayStyle = isDataLoading || isNaN(selection[0]) || isNaN(selection[1]) ? "none" : null;
```
 it can replace the duplicate code in `line 350`